src/AccessibilityInsights.Win32Tests/Win32ApisTests.cs

```diff
@@ -263,5 +263,47 @@ public void IsWindowsRS5OrLater_BuildValueIsGreaterThanTarget_ReturnsTrue()
                 Assert.AreEqual(2, index);
             }
         }
+
+        [TestMethod]
+        public void GetCurrentWindowsVersionForTelemetry_CurrentVersionDoesNotExist_ReturnsNull()
+        {
+            using (ShimsContext.Create())
+            {
+                List<string> returnValues = new List<string> { "" };
+                int index = 0;
+                ShimRegistry.GetValueStringStringObject = (_, __, def) => (++index > returnValues.Count ? def : returnValues[index - 1]);
+
+                Assert.IsNull(NativeMethods.GetCurrentWindowsVersionForTelemetry());
```

Comment:
Not that I expect any change in this PR, but isn't the NativeMethods class supposed to only be for pInvoke signatures of native methods?

Reply:
No longer an issue in the revised format

---
In reply to: [277719578](https://github.com/Microsoft/accessibility-insights-windows/pull/332#discussion_r277719578) [](ancestors = 277719578)